index.js

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs';
+
+const data = fs.readFileSync('words_alpha.txt', 'utf8');
+//console.log(data);
+const words = data.split('\r\n');
+//console.log(words);
+/*let longestWord = words.reduce(function(a, b) { 
+  return a.length > b.length ? a : b
+}, '');*/
+// console.log(longestWord, words.indexOf(longestWord));
+
+
+
+
+
+
+const ALPHABET = ['a', 'b', 'c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'];
+
+//console.log(isAlphabetical("acbde"))
+
+
+function focusOnWords(arr){ // We pass in an array of words 
+
+    let result;
+    let foundWords = []; //This is the array that'll list the words that got their letters in alphabetical order
+
+    for(let i = 0; i < arr.length; i++){ // We loop through the words inside the array
+        result = focusOnLetters(arr[i]); //Now go to the next functions to understand what goes on
+        if(result != 'not in Alphabetical Order'){
+            /*We're back to check if the letters of a given word are in alphabetical order,
+            if they are, we wanna add the word to the foundWords array*/
+            foundWords.push(result)
+        }
+    }
+
+    return foundWords
+}
+
+function focusOnLetters(word){ //This function focuses on each word of the array
```

Comment:
you don't need many of the comments here. the function and variable names should give a good idea what the code does. try to use comments only for the "why" not the "how"